helper/fairshare/jobmanager.go

```diff
@@ -243,6 +243,9 @@ func (j *JobManager) incrementWorkerCount(queueID string) {
 	defer j.l.Unlock()
 
 	j.workerCount[queueID]++
+	if j.metricSink != nil {
+		j.metricSink.IncrCounterWithLabels([]string{"expire", "queue_workers"}, 1, []metrics.Label{{"queue_id", queueID}})
```

Comment:
Since the fair sharing layer is currently general-purpose we probably shouldn't be assuming it's always used for expiration (as this metric does). What does that mean? One option would be to keep it in line with other fair share metrics (`...[]string{"job_manager", "workers"}...`) and then update documentation on how to get the data in the case of expiration. Thoughts @briankassouf ?

Reply:
ah yes this makes sense, will get the name in line with the other metrics in this file!

Reply:
2558e12